content/en/synthetics/private_locations.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Run Synthetics tests from Private Locations
+kind: documentation
+description: Run Synthetics API and browser tests from Private Locations
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+<div class="alert alert-warning">
+This feature is in beta. <a href="https://app.datadoghq.com/privatelocations/2019signup">Sign up for the beta</a> to enable Synthetics private locations for your account.
```

Comment:
Link to form is different if EU instances - should we specify both?

Reply:
well spotted.